Octokit/Models/Response/Team.cs

```diff
@@ -71,9 +72,14 @@ public Team(string url, string htmlUrl, int id, string nodeId, string slug, stri
         public StringEnum<TeamPrivacy> Privacy { get; private set; }
 
         /// <summary>
-        /// permission attached to this team
+        /// Deprecated. The permission that new repositories will be added to the team with when none is specified
         /// </summary>
-        public StringEnum<PermissionLevel> Permission { get; private set; }
+        public string Permission { get; private set; }
```

Comment:
Apparently, you can create custom roles with github enterprise and use them to set team permission. In the spec the permission is modeled as a string rather than an enum, so I modeled the Permission as a string.

Reply:
Understood, thank you.